abm/projects/cooperative_signaling/cs_simulation/cs_sims.py

```diff
@@ -170,6 +160,35 @@ def add_new_agent(self, id, x, y, orient, with_proove=False,
                 self.agents.add(agent)
                 agent_proven = True
 
+    def create_agents(self):
+        """
+        Creating agents according to how the simulation class was initialized
+        """
+        for i in range(self.N):
+            orient = np.random.uniform(0, 2 * np.pi)
+            dist = np.random.rand() * (
+                    self.HEIGHT / 2 - 2 * self.window_pad - self.agent_radii)
+            x = np.cos(orient) * dist + self.WIDTH / 2
```

Comment:
for me this still yield new agents outside of the circular arena when I am increasing the agent number in playground tool, this is probably in cs_isims.py that we need to ovverride

Reply:
Nice, I missed that bug! This should solve it: 12a35dca8d530c28b595b8ac11eee8898ee10eb2